modules/tinymce/src/core/main/ts/api/EditorCommands.ts

```diff
@@ -221,371 +162,41 @@ class EditorCommands {
    * @return {Boolean} true/false if the command is supported or not.
    */
   public queryCommandSupported(command: string): boolean {
-    command = command.toLowerCase();
-
-    if (this.commands.exec[command]) {
-      return true;
+    if (this.editor.quirks.isHidden() || this.editor.removed) {
+      return false;
     }
```

Comment:
This seems wrong? Whether the editor is hidden or removed seems irrelevant to a boolean check of whether a command string exists on an object.

This breaks our Table Model branch, which needs to switch from `hasCustomCommand` to `queryCommandSupported` before `editor.quirks` is created.

Reply:
Oops yes that is wrong it should only check if the editor is removed. But it depends also on how we interpret the function of that should it be is the command registered or is the command available for execution if the editor is removed you can't execute it. But calling commands on removed editors seems like an edge case. But calling it on editors that doesn't have a selection is fairly common. I can make a follow up ticket to this but maybe remove the is removed check so that it is a is the command registered instead.

Reply:
I deleted the entire if block on the model PR. If editors that don't have a selection are  problem, perhaps we need further checks here (or better documentation to explain that "supported" commands need further checks to see if they're going to function correctly).

Reply:
Yeah we should remove the entire block since we want it to the equal to the hasCsutomCommand and checking for commands on a removed editor is sich an edge case. We need to update the changelog and the code should I still do a separate ticket and or for this or will you just bake it into your branch?

Reply:
I was trying to say we already have :)
https://github.com/tinymce/tinymce/pull/7583/commits/aaf1231977f0a3054b3f444b30ce6aaabf5b9071

We forgot to do anything with the changelog though.